.github/workflows/test.yml

```
@@ -0,0 +1,25 @@
+name: Test
+on:
+  push:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
+  pull_request:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
```

```suggestion
    paths:
      - 'src/**'
      -  action.yml
      -  '.github/wiki/**'
      - '.github/workflows/test.yml'
  pull_request:
    branches: [main]
    paths:
      - 'src/**'
      -  action.yml
      -  '.github/wiki/**'
      - '.github/workflows/test.yml'
```
Just trying something out quickly